src/global-messages/GlobalMessage.tsx

```diff
@@ -4,23 +4,26 @@ import {useGlobalMessagesState} from '@atb/global-messages/GlobalMessagesContext
 import {MessageBox} from '@atb/components/message-box';
 import {StyleProp, ViewStyle} from 'react-native';
 import {
-  GlobalMessageContextType,
+  GlobalMessageContextEnum,
   GlobalMessageType,
 } from '@atb/global-messages/types';
 import {getTextForLanguage} from '@atb/translations';
 import {useNow} from '@atb/utils/use-now';
 import {isWithinTimeRange} from '@atb/global-messages/is-within-time-range';
+import {RuleVariables} from './rules';
 
 type Props = {
-  globalMessageContext?: GlobalMessageContextType | 'all';
+  globalMessageContext?: GlobalMessageContextEnum | 'all';
```

Comment:
Yes I don't think it makes sense anymore 🤔 `'all'` is only used in ServiceDisruptionSheet, which right now kind of works, by not showing messages that have rules (because no `ruleVariables` are provided). But that surely isn't how we want them to actually work.

It could be solved by using arrays for contexts (everywhere) like this:

```typescript
<GlobalMessage
    globalMessageContexts={[
      GlobalMessageContextEnum.appTicketing,
      GlobalMessageContextEnum.appAssistant,
      GlobalMessageContextEnum.appDepartures,
    ]}
    includeDismissed={true}
  />
```

Alternatively, we could add a new context "app-all" or "app-service-disruptions"that needs to be set in Firestore for it to show up in service disruptions, but that might be easy to forget when publishing messages.

What do you think?

Reply:
Landed on adding a context `app-service-disruptions`. There might be some changes to the service disruptions list in the future, so we decided to not make this a bigger task than it needs to be.